### DISCUSSION TITLE
Deno support

### DISCUSSION THREAD

**Discussion by italodeandra** [typegoose/mongodb-memory-server#685] [General]
Hi,

I currently use `mongodb-memory-server` on all my Node projects that are using MongoDB. But I'm slowly moving towards Deno.

I tried using `mongodb-memory-server` on Deno with Deno's compatibility mode but with no success since it uses some packages specific for Node (like `new-find-package-json` down in the peer deps).

Do anyone want to go into this adventure of supporting this lib on Deno?

**Comment by hasezoey** [comment]
"Deno" certainly looks interesting, though i currently do not plan to add explicit support, but you could maybe point out some things to make it more compliant (and nodejs will still be the priority)

also as a side note, this module will not switch to ESModules until a major version and all tools support it (currently the lowest required nodejs version works with it, typescript is still a little flaky, and jest has not stable support yet)

> like new-find-package-json down in the peer deps

package `new-find-package-json` is a normal dependency, not a peer-dependency, also if there is a problem with it, you can also add it to the list of "making it more compliant", because it is a package i also maintain (since the original one was not great)
